_posts/2017-03-23-first-ocim-blog.markdown

```diff
@@ -0,0 +1,107 @@
+
+Following Martin's talk, Hannes Voigt from the Technical University of Dresden walked us through his research project, in which also Michael Hunger from Neo had participated.
+The topic was virtual graphs and views, and featured several intriguing extensions to Cypher, which were expressed as able to cross the concept chasm, which Hannes explained as the different levels of abstraction that users view their data in.
+At the lowest level is the pure data, big and raw.
+At higher levels, larger patterns start to appear, which are made up of groups of nodes and relationships from the lower levels.
+These larger patterns are in the model constructed using virtual nodes and relationships, with the additional ability to define views which provide several interesting qualities, such as performance optimizations and query modularization.
+
+---
+
+The afternoon session was featured two larger sections in which the members of the Cypher Language Group, Neo's internal team responsible for developing Cypher, presented views and ideas on how to address the most prominently mentioned shortcomings of the language in its current form.
+Mats Rydberg presented a proposal for a new schema/constraint syntax, and also talked more in-depth on the Technology Compatibility Kit, highlighting its usefulness to verify that a multitude of language implementations are semantically consistent.
+Petra Selmer went through the latest thinking on several classes of subqueries, including syntax proposals.
+Petra also detailed the revised Cypher improvement process, which has been designed to fit the open, collaborative format intended for the openCypher project.
+As mentioned above, Stefan and Oskar discussed semantics of pattern matching, in terms of isomorphism/homomorphism of subgraphs and entities.
+Tobias Lindaaker provided insights in how Cypher could complete its support for Conjunctive Regular Path Queries (CRPQs), going through historical thinking as well as recent syntax and semantics proposals.
+The topic of vendor-specific extensions, including some of the pitfalls that should be looked out for from experiences with the SQL standard was also presented by Tobias.
+
+The last session also featured Paolo Guagliardo from the University of Edinburgh, who gave us a look into a recent research project on formalising semantics of SQL, and what advantages that a language may be given throug a formal specification.
+Paolo also announced the newly started project of producing a formal semantics specification of Cypher, which is to be carried out by his research team, including Nadime Francis and prof. Leonid Libkin.
+This project will be carried out during the spring of 2017, and reports of progress will be given at upcoming oCIMs.
+
+## Some reflections
+
+Overall, the meeting was received in a very positive light.
```

Comment:
"Overall, the meeting was received in a very positive light."
->
"All in all, the meeting was a resounding success, and we look forward to many more in the future."